packages/editor/src/bindings/post-meta.js

```diff
@@ -142,6 +142,32 @@ export default {
 		return true;
 	},
 	getFieldsList( { select, context } ) {
+		// Deprecated, will be removed after 6.9.
```

Comment:
Should it include a deprecation warning? I would be surprised if it was possible to use it, but it's better to keep it for some time.